src/ioctls/vcpu.rs

```diff
@@ -1378,6 +1378,11 @@ impl VcpuFd {
             Ok(())
         }
     }
+
+    /// Get reference to the underlying vcpu file object and then get access to underlying RawFd.
```

Comment:
This comment does not look accurate. It just returns a reference to the underlying vCPU file. Can you update this comment to something like:
```
\\\ Get a reference to the underlying vCPU file.
\\\
\\\ This can be used in the context of CPU hotplug for ..... todo details here why is this actually needed.

Reply:
nit: the commit message is not accurate as we're not exporting `new_vmfd`.

Reply:
dropped the patch, it's not needed anymore:)